.github/workflows/ftr-e2e-dashboards-observability-test.yml

```diff
@@ -128,12 +128,21 @@ jobs:
         if: ${{ runner.os == 'Linux' }}
         run: |
           cd ./OpenSearch-Dashboards
-          if timeout 600 grep -q "bundles compiled successfully after" <(tail -n0 -f dashboard.log); then
-            echo "OpenSearch Dashboards compiled successfully."
-          else
-            echo "Timeout for 600 seconds reached. OpenSearch Dashboards did not finish compiling."
-            exit 1
-          fi
+          timeout=900
```

Comment:
Let's revert this as it is no longer necessary thanks to @Swiddis

Reply:
reverted. It's great, we don't need to wait long time to get OSD ready.